sdk/eventgrid/azure-messaging-eventgrid/swagger/main.tsp

```diff
@@ -0,0 +1,295 @@
+import "@typespec/http";
```

Comment:
Do we need to check this in? Do we not have the TypeSpec files in rest-api repo?

Reply:
will resolve after beta after offline chat.